src/screens/Disputes/DisputesUtils.res

```diff
@@ -136,6 +136,17 @@ let initialFixedFilter = () => [
   ),
 ]
 
+let getLabelFromFilterType = (filter: filter) => {
```

Comment:
can be written direct , wont be reauired a function cause its not doing any other thing rather then one line string conversion

Reply:
i think we can keep this because it makes it clear the diff between the label and values of the filters, since we are doing `Array.push(filterType)` at many places (for additional filters) so using this over there